hs_core/models.py

```diff
@@ -2251,21 +2284,7 @@ def get_citation(self):
         else:
             return CITATION_ERROR
 
-        ref_rel = self.metadata.relations.all().filter(type='isHostedBy').first()
```

Comment:
Why was this block removed?

Reply:
Developers are perhaps most accustomed to having GitHub Issues to review for the functional requirements and use cases for a Pull Request. In this case, a google document has substituted for this step. Please find attached https://docs.google.com/document/d/1Rn72NMEo2QSErzoC6KPItp5bKjRwbqed8GWYDBqlHwg